test-suite/squidconf/bad-regex.conf.instructions

```diff
@@ -0,0 +1,2 @@
+expect-failure (ERROR|FATAL):.*POSIX.regcomp.*failure
```

Comment:
These patterns should only match the log output produced by the code tested.

For the test enabled by this PR the expected output is:
```
ERROR: configuration failure: POSIX regcomp(3) failure: (13) Invalid preceding regular expression
    regular expression: *
```

```suggestion
expect-failure ERROR:.configuration.failure:.POSIX.regcomp.*failure
```

Reply:
I committed your suggestion (fdec41e1404566f771181d2d778da6df6c77c002).

> These patterns should only match the log output produced by the code tested.

No constant regular expression can satisfy the above criteria: No mater what regex one writes, I can always invent a future Squid code variation where that regex will match something it should not (or not match something it should). FWIW, the originally proposed expression only matched the intended Squid output, but also allowed for highly anticipated Squid improvement where all fatal configuration ERRORs will become labeled as FATAL. I think that was a slightly better approach, but I (obviously) do not insist on it.